### PR TITLE
Update steward to v0.2.2

### PR DESCRIPTION
**Pull request by ccremer**
## Checklist

- [x] Keep pull requests small so they can be easily reviewed.
- [x] Update the documentation.
- [x] Update the ./CHANGELOG.md.
- [ ] Link this PR to related issues.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -72,3 +72,4 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 [#84]: https://github.com/projectsyn/lieutenant-api/pull/84
 [#86]: https://github.com/projectsyn/lieutenant-api/pull/86
 [#90]: https://github.com/projectsyn/lieutenant-api/pull/90
+[#103]: https://github.com/projectsyn/lieutenant-api/pull/103
```
Comment (corvus-ch) [inline CHANGELOG.md:75]: Missing the line in the Unreleased section

**Review by corvus-ch — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## Unreleased
+
+- Update steward to v0.2.2([#103])
```
Comment (srueg) [inline CHANGELOG.md:9]:
```suggestion
- Update steward to v0.2.2 ([#103])
```

**Review by srueg — APPROVED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## Unreleased
```
Comment (corvus-ch) [inline CHANGELOG.md:7]:
```suggestion
## [Unreleased]
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
...
+
+- Update steward to v0.2.2 ([#103])
```
Comment (corvus-ch) [inline CHANGELOG.md:9]:
```suggestion
### Changed
- Update steward to v0.2.2 ([#103])
```

**Review by corvus-ch — CHANGES_REQUESTED** [review]

**Review by corvus-ch — APPROVED** [review]

**Pull request merged**